department-of-veterans-affairs/vets-api
# [44619] Make Sign in Service user_uuid equal to csp_uuid for now, to …

…prevent thrash with SiS release

## Description of change
This PR makes it so that the User created when authenticating with Sign in Service will use either a logingov_uuid or idme_uuid as the user.uuid, instead of the UserAccount.id. We are still planning on migrating user.uuid to user_account, but we will do so separately from the Sign in Service release

## Original issue(s)
department-of-veterans-affairs/va.gov-team#44619

## Things to know about this PR
- To test, authenticate with sign in service
- Call introspect route, and make a note of the `uuid` value
- In Rails console, confirm if this is the `idme_uuid` or `logingov_uuid` of the user that authenticated (`User.find(<uuid>).user_verification` should show a `UserVerification` with an `idme_uuid` or `logingov_uuid` value that should match)
- Also, create an EVSS claim if possible, then authenticate as SSOe, you should be able to see the EVSS claim
